generate-examples/generate-cert-chain.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+# this script generates a 3-cert ECDSA P-256 chain (root -> CA -> issuer) valid for 5 years
+
+# directory where intermediate files are kept
+tmpdir=certs
+mkdir -p $tmpdir
+# directory where generated JWK files are stored
+outdir=certs
```

Comment:
If this dir is `.gitignore`d, how do we maintain stable public certs for demos? Might be good to follow the pattern of putting generated certs in `src/config/` (alongside generated private keys) -- or else revamp both together.

Reply:
Yes, final JWKS with x5c will be in `src/config`. I'm writing code now to update the issuer set in there. openssl generates some intermediate files we don't need. Final certs and keys might be saved in `src/config` as well.